src/main/webapp/i18n/de/student-dashboard.json

```
@@ -47,6 +47,12 @@
             },
             "menu": {
                 "exercises": "Aufgaben",
+                "textExercise": "Text Aufgabe",
+                "fileUploadExercise": "File Upload Aufgabe",
+                "programmingExercise": "Programming Aufgabe",
+                "modelingExercise": "Modeling Aufgabe",
+                "quizExercise": "Quiz Aufgabe",
+                "repository": "Repository",
```

Why are we introducing new translations here? Why are we not re-using the translations from `global.json`?

```suggestion
                "textExercise": "Textaufgabe",
                "fileUploadExercise": "Datei-Upload-Aufgabe",
                "programmingExercise": "Programmieraufgabe",
                "modelingExercise": "Modellierungsaufgabe",
                "quizExercise": "Quiz Aufgabe",
                "repository": "Repository",
```

---

Here are all the headings for the new student view. This will make it easier to iterate generically.